### PR TITLE
BZ1990953: Only the first container can use SR-IOV network attachment

### PR DESCRIPTION
**Pull request by jboxman-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1990953

### CODE REVIEW

**Comment by jboxman-rh** [comment]
@zshi-redhat, I re-opened this under a separate PR.

**Comment by zshi-redhat** [comment]
> @zshi-redhat, I re-opened this under a separate PR.

Thanks @jboxman !

/lgtm

**Comment by zshi-redhat** [comment]
PR title may need to change from `BZ-1990953` to `Bug1990953`

File: modules/nw-multus-add-pod.adoc [hunk 1]
```diff
@@ -29,7 +29,11 @@ The pod must be in the same namespace as the additional network.
 ifdef::sriov[]
 [NOTE]
 =====
-If a network attachment is managed by the SR-IOV Network Operator, the SR-IOV Network Resource Injector adds the `resource` field to the `Pod` object automatically.
+The SR-IOV Network Resource Injector adds the `resource` field to the first container in a pod automatically.
+
+If you are using an Intel network interface controller (NIC) in Data Plane Development Kit (DPDK) mode, only the first container in your pod is configured to access the NIC. Your SR-IOV additional network is configured for DPDK mode if the `deviceType` is set to `vfio-pci` in the `SriovNetworkNodePolicy` object.
+
+You can workaround this issue by either ensuring that the container that needs access to the NIC is the first container defined in the `Pod` object or by disabling the Network Resource Injector. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1990953[BZ#1990953].
```
Comment (bergerhoffer) [inline modules/nw-multus-add-pod.adoc:36]: I believe "workaround" is a noun only. If you want to make it a verb, I'd say "You can work around"

**Review by bergerhoffer — COMMENTED** [review]
Just one wording thing

File: modules/nw-multus-add-pod.adoc (hunk 1 above)
Comment (jboxman-rh) [reply modules/nw-multus-add-pod.adoc:36]: @bergerhoffer oops thanks!

**Pull request merged**